helpers/main_helper.py

```diff
@@ -0,0 +1,75 @@
+import random, string, itertools
+
+# English words searched in a grid
+words = open('words.txt', 'r').read().splitlines()
+# All low case letters from English alphabet
+letters = string.ascii_lowercase
+found_words = set()
+
+
+def create_grid(a, b):
+	"""Create grid of letters with a x b dimensions"""
+	if a < 1 or b < 1: raise Exception('Dimensions of grid can\'t be less than 1')
+	global grid_letters
+	grid_letters = [list(map(lambda let: random.choice(letters), range(a))) for _ in itertools.repeat(object, b)]
+
+
+def grid_transpose(b):
+	"""Transpose grid"""
+	global transpose_letters
+	transpose_letters = [row[i] for i in range(len(grid_letters[0])) for row in grid_letters]
```

Comment:
is this just making a copy of the original `grid_letters`?

Reply:
not quite, its like arr.flatten in ruby (did not know if the same exist in python)
```ruby
[["g", "k", "x", "m"], ["e", "r", "p", "j"], ["j", "s", "x", "c"], ["f", "z", "j", "q"]]
["q", "j", "z", "f", "c", "x", "s", "j", "j", "p", "r", "e", "m", "x", "k", "g"]
```